config/samples/infrastructure_v1alpha1_dockermachine.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
```

Comment:
Might be good to exclude `config/samples/` and add to .gitignore, since they aren't valid examples.

Reply:
oh, good call, will do ~ ty